server/handlers.go

```diff
@@ -259,16 +259,14 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 	}
 
 	connectorInfos := make([]connectorInfo, len(connectors))
-	i := 0
-	for _, conn := range connectors {
-		connectorInfos[i] = connectorInfo{
-			ID:   conn.ID,
```

Comment:
@nabokihms Further thinking about this: How about leaving the ID here? The templates can be replaced and someone might depend on the ID in a custom template. The default templates can continue using the type.

Reply:
It seems ok for me. I will fix it.